src/utils/index.js

```diff
@@ -4,4 +4,8 @@ export const contains = (list, value) => {
   return list.findIndex(item => equals(item, value)) > -1;
 };
 
+export const determinePropertyMessage = (actual, property, message = 'Not Accessible') => {
```

Comment:
I couldn't inline this logic because jest didn't recognize the coverage which I provided by testing the error message itself so I thought that I would generalize it and test it that way. What do you think?